src/build.js

```diff
@@ -30,7 +34,8 @@ module.exports = function build(markdown) {
   return `
 ${doImports}
 
-module.exports = function() {
+export const attributes = ${JSON.stringify(camelize(frontMatterAttributes))};
+export default function() {
   return (
     <div>
```

Comment:
I'm pretty sure that this is part of the string that will also get parsed by babel. How do you suggest I proceed? Would you prefer to see:

```javascript
module.exports = function () { /* component */ };
module.exports.attributes = {/* front matter attributes */};
```

I don't love this approach because I'm afraid of eventually conflicting with some react specific  static attributes, because this puts `attributes` on the same level as `propTypes` and others.